clients/ptranslator/src/test/java/org/hawkular/metrics/clients/ptrans/fullstack/CollectdITest.java

```diff
@@ -264,6 +268,7 @@ private Point collectdLogToPoint(String line) {
             String type = split[split.length - 1];
 
             urlConnection = (HttpURLConnection) new URL(findGaugeDataUrl(metricName)).openConnection();
+            urlConnection.setRequestProperty("tenantId", tenant);
```

Comment:
I think it is a bit confusing to me that you sometimes use `TENANT` and sometimes `tenantId`. While I see that the former is used for the config properties, it still confuses me. Perhaps not using a static import would help me here.

Btw: why did you introduce `getExternalForm()` and not override `toString()`?